Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -831,6 +831,8 @@ - (void)testSpecTests {
   }
 
   // Now iterate over them and run them.
+  __block NSUInteger testPassCount = 0;
```

Comment:
Avoid platform dependent types like `NSUInteger` unless you specifically need them to interact with Objective-C APIs. This can just be `int`.

Reply:
Done.